include/session/types.hpp

```diff
@@ -1,13 +1,23 @@
 #pragma once
 
+#include <array>
 #include <cstdint>
-#include <span>
-#include <string>
-#include <string_view>
-#include <vector>
 
-namespace session { namespace config {
+namespace session {
+using array_uc32 = std::array<std::uint8_t, 32>;
+using array_uc33 = std::array<std::uint8_t, 33>;
+using array_uc64 = std::array<std::uint8_t, 64>;
 
-    using seqno_t = std::int64_t;
+enum class SessionIDPrefix {
+    standard = 0,
+    group = 0x3,
+    community_blinded_legacy = 0x5,
+    community_blinded = 0x15,
+    version_blinded = 0x25,
+    unblinded = 0x7,
+};
```

Comment:
I am very confused by this. Is it supposed to map to real prefixes?
If yes this should all be changed a fair bit:
```hpp
```suggestion
enum class SessionIDPrefix {
    standard = 0x05,
    group = 0x03,
    community_unblinded = 0x05,
    community_blinded_legacy = 0x15,
    community_blinded = 0x25,
    version_blinded = 0x07,
};
```

I guess 0x3 is actually 0x03, but the prefixes/names associated are not correct too

Reply:
The `community_unblinded` uses a `00` prefix which represents the users ed25519 pubkey (and a signature signed with that) whereas the `05` prefix would be for the users x25519 pubkey (and it's signature)

It's used for the SOGS signature header when blinding is disabled - for reference there is a function to get the string prefix for this enum in `internal.hpp`:
```
inline constexpr std::string_view to_string(session::SessionIDPrefix prefix) {
    switch (prefix) {
        case session::SessionIDPrefix::unblinded: return "00"sv;
        case session::SessionIDPrefix::group: return "03"sv;
        case session::SessionIDPrefix::standard: return "05"sv;
        case session::SessionIDPrefix::community_blinded_legacy: return "15"sv;
        case session::SessionIDPrefix::community_blinded: return "25"sv;
        case session::SessionIDPrefix::version_blinded: return "07"sv;
    }

    return "05"sv;  // Fallback to standard, shouldn't occur
};
```